dotnetv3/S3/CrossRegionReplicationExample/CrossRegionReplication.cs

```diff
@@ -1,36 +1,36 @@
 ﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX - License - Identifier: Apache - 2.0
 
+/// <summary>
+/// This example shows how to configure replication for the objects in an
+/// Amazon Simple Storage Service (Amazon S3) bucket across AWS regions.
+/// It was created using AWS SDK for .NET version 3.7 and .NET Core 5.0.
+/// </summary>
 namespace CrossRegionReplicationExample
 {
+    // snippet-start:[S3.dotnetv3.CrossRegionReplicationExample]
     using System;
     using System.Threading.Tasks;
     using Amazon;
     using Amazon.S3;
     using Amazon.S3.Model;
 
-    /// <summary>
-    /// This example shows how to configure replication for the objects in an
-    /// Amazon Simple Storage Service (Amazon S3) bucket across AWS regions.
-    /// It was created using AWS SDK for .NET version 3.7 and .NET Core 5.0.
-    /// </summary>
     public class CrossRegionReplication
     {
         public static async Task Main()
         {
             string sourceBucket = "doc-example-bucket1";
 
             // The Amazon Resource Name (ARN) of the Amazon S3 destination
-            // bucket. The included value is only an example. Be sure to change
-            // this to the ARN of your destination bucket.
+            // bucket.
             //
             // The role ARN should be the ARN of a role that can read and
-            // write to an S3 bucket.
+            // write to an Amazon S3 bucket.
             //
-            // The examples below are not valid ARNs and must be replaced with
-            // valid ARNS from your account.
+            // The following values are not valid ARNs. You should replace them
+            // with valid ARNS from your account.
```

Comment:
ARNS -> ARNs
(just for intelligibility)

Reply:
Fixed.